crates/biome_js_analyze/src/semantic_analyzers/suspicious/no_console_log.rs

```diff
@@ -71,4 +75,18 @@ impl Rule for NoConsoleLog {
             }),
```

Comment:
We could add a note saying that if it is not for debugging purpose, `console.info` is more appropriate?